rust/ql/consistency-queries/DataFlowConsistency.ql

```diff
@@ -14,3 +14,11 @@ private import codeql.dataflow.internal.DataFlowImplConsistency
 private module Input implements InputSig<Location, RustDataFlow> { }
 
 import MakeConsistency<Location, RustDataFlow, RustTaintTracking, Input>
+
+/**
+ * This adds `AstNode` class names to the results of `uniqueNodeToString`, if any.
+ */
+query predicate uniqueNodeToStringClasses(Node n, string cls) {
```

Comment:
Perhaps do this in `AstConsistency.qll` instead (replacing the existing `multipleToStrings` check with a `uniqueToString` check)? That way it will be checked for all AST nodes, not just data flow nodes.

Reply:
makes sense!